lib/main.dart

```diff
@@ -117,9 +118,12 @@ Future<void> createIconsFromArguments(List<String> arguments) async {
         );
       }
       print('\n✓ Successfully generated launcher icons for flavors');
-    } catch (e) {
+    } catch (e, st) {
       stderr.writeln('\n✕ Could not generate launcher icons for flavors');
       stderr.writeln(e);
+      if (logger.isVerbose) {
+        stderr.writeln(st);
+      }
       exit(2);
```

Comment:
Out of scope

If you want to submit a PR for increasing the amount of logs when verbose mode is activated feel free to add this later